<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMessage</class>
 <widget class="QWidget" name="AdminMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnSeeAll">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>查看所有用户信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChagePassword">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>修改用户密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChageEmail">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>设备管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChageEmail_3">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>告警</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChageEmail_4">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>数据监控</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChageEmail_5">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>数据分析</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteUser">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>删除用户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChageEmail_2">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>日志</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnclose">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
